libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/SyncManager.java

```diff
@@ -287,6 +286,58 @@ public SyncState syncUp(SyncUpTarget target, SyncOptions options, String soupNam
     	return sync;
     }
 
+    /**
+     * Deletes local copies of records that have been deleted on the server.
+     *
+     * @param syncId Sync ID.
+     * @throws JSONException
+     */
+    public void syncRemoteDeletes(long syncId) throws JSONException {
```

Comment:
It will only remove local records that shouldn't be there. It's not
bringing down updates or records. Maybe it should be called:
cleanReSyncGhosts?